### ISSUE TITLE
Wrong domains listed in newly opened tabs

### ISSUE THREAD

**Issue by hevp** [compvid30/scriptblock#3]
The wrong domains are listed when a link is opened in a new tab. In fact, the domains are listed of the neighboring tab (left-side). There is no way to prevent this from happening, except closing that neighboring tab.

**Comment by slick9115** [comment]
Agreed. I can't find out how to reproduce this, but it happens from time to time.

**Comment by StefanFabian** [comment]
I think it happens when you get forwarded.
Because every time I click on a link on facebook it only shows facebook as a domain in the script blocker.
So I can't unblock the scripts from the actual page.

**Comment by compvid30** [comment]
I'am a bit low on time to check this issue out at the moment. As soon as I find some time, I'll look for it. If anyone else finds the bug, please let me know.

**Comment by hevp** [comment]
I think forwarding is the problem indeed. Opening the same URL in a new tab solves it for now.

**Comment by ingorichter** [comment]
Forwarding wouldn't explain what I see. It seems, that for every open tab, there is a random domain mentioned. It's not necessarily the neighboring tab. It's any of the open tabs.

**Comment by niklasschaffrath** [comment]
Same problem here, after following link. Link has nothing special, just a simple html "a" tag. Reloading the page doesn't help, copy/paste to new tab does help.

**Comment by compvid30** [comment]
Hello

this looks for me at the moment like a bug in chrome. Can you please post the chrome version where this happens.

**Comment by ingorichter** [comment]
I'm running `45.0.2454.93 (64-bit)` on OSX

**Comment by niklasschaffrath** [comment]
I just got updated to 45.0.2454.93 (64-bit) (OSX).
But haven't seen the problem with this version yet, just the previous one.
I will look out if it happens again when enough tabs are open.

**Comment by compvid30** [comment]
I tested Version 45.0.2454.93 m on Windows and all works fine. Seems to be a bug in one or more Chrome versions. Please update your Chrome.

**Comment by hevp** [comment]
With version 45.0.2454.93 for OS X the problem no longer occurs.

**Comment by compvid30** [comment]
As I thought, it was a bug in the chromium code.

**Comment by ingorichter** [comment]
Chrome was updated to `Version 45.0.2454.99 (64-bit)`. I will have an eye on it. Thanks.
